modules/kubecf/gen_config.sh

```diff
@@ -32,6 +32,13 @@ else
 INGRESS_BLOCK=''
 fi
 
+last_worker=$(kubectl get nodes | tail -1 | cut -f 1 -d ' ')
+num_cores=$(kubectl describe node "$last_worker" | \
+          grep Allocatable -A 2 | grep cpu | tr -s ' ' | cut -f 3 -d ' ')
+if [ "$CATS_NODES" == corenumber ]; then
```

Comment:
[MINOR] Maybe move the 2 lines above inside the `if`? This will save some time from describing and getting the nodes when it's not needed.

Reply:
Doh, true! done.